src/scenes/Activities/ActivityHistory/EventRows/index.js

```diff
@@ -58,10 +58,11 @@ class EventRow extends Component {
     }
 
     render() {
-      const { transaction, intl, classes } = this.props;
-      const { txType, txid, txReceipt: { cumulativeGasUsed }, txStatus, block, name, address, amount } = transaction;
+      const { transaction, intl, classes, store: { wallet: { exchangeRate } } } = this.props;
+      const { txType, txid, txReceipt: { cumulativeGasUsed, gasPrice }, txStatus, block, name, address, amount } = transaction;
       const blockTime = block ? block.blockTime : messages.strPendingMsg;
       const { expanded } = this.state;
+      const gasFee = (satoshiToDecimal(gasPrice) / weiToDecimal(exchangeRate)) * satoshiToDecimal(cumulativeGasUsed);
```

Comment:
if the tx is unconfirmed, cumulativeGasUsed will be undefined but gasUsed will be defined. 

If tx is confirmed, you can use cumulative.